### PR TITLE
N17: Undergrad Thesis on game scripting for legion

### PR DESCRIPTION
**Pull request by pablodiegoss**
Our section of #408 :D

### CODE REVIEW

File: content/posts/newsletter-017/index.md [hunk 1]
```diff
@@ -87,6 +87,32 @@ This month's updates:
 
 ## Learning Material Updates
 
+### [Undergrad Thesis on game scripting for Legion]
+
+In december 2020, [@pablodiegoss] and [@rodrigocam] presented their undergrad
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-017/index.md:92]:
```suggestion
In December 2020, [@pablodiegoss] and [@rodrigocam] presented their undergrad
```

File: content/posts/newsletter-017/index.md [hunk 2]
```diff
@@ -87,6 +87,32 @@ This month's updates:
...
+thesis where they created a proof-of-concept scripting system for the Legion ECS
+, called [Legion Script]. Legion is being currently integrated into the
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-017/index.md:94]:
```suggestion
thesis where they created a proof-of-concept scripting system for the Legion ECS,
called [Legion Script]. Legion is being currently integrated into the
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Looks good, just a few minor formatting things:

**Comment by pablodiegoss** [comment]
> Looks good, just a few minor formatting things:

Awesome, I've just incorporated that feedback, thanks! :D

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by ozkriff** [comment]
Made a few style & fmt changes for consistency with the rest of the newsletter. Also, reworded the link to the forum thread.

@pablodiegoss and @rodrigocam are you ok with these changes?

**Comment by rodrigocam** [comment]
It seems great to me!

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Thanks for the PR! Good luck with the upcoming blog posts!

**Pull request merged**